addon/mixins/sl-tooltip-enabled.js

```diff
@@ -123,8 +138,6 @@ export default Ember.Mixin.create({
 
         // First-time rendering
         if ( 'undefined' === Ember.typeOf( originalTitle ) ) {
-            this.set( 'dataToggle', 'tooltip' );
```

Comment:
The `dataToggle` value never needs to be changed based on the value(s) of `popover` and `title`?